chromium_src/components/permissions/request_type.cc

```diff
@@ -39,6 +39,7 @@ constexpr auto& kMicIconValue = vector_icons::kMicIcon;
   case RequestType::kBraveSolana:                    \
   case RequestType::kBraveGoogleSignInPermission:    \
   case RequestType::kBraveLocalhostAccessPermission: \
+  case RequestType::kBraveDefault:                   \
```

Comment:
comments hidden above that are no longer correct 

<img width="527" alt="image" src="https://github.com/brave/brave-core/assets/34129/3e762f4e-ab85-4509-aa58-51d697045f46">

Reply:
yeah, right, I removed it.